### ISSUE TITLE
suggestion to add reference to pip

### ISSUE THREAD

**Issue by ggrrll** [jaidevd/pyhht#49]
Hi, 

thanks for the package -- I just started to use it.

I was thinking that would be useful for a quick start to see the reference to pip-installation in the readme https://pypi.org/project/pyhht/, even if it's a trivial thing for python users (maybe not for others :))

**Comment by jaidevd** [comment]
Hey @ggrrll 

Yes, there should be a reference about pip installation in the README itself, but the version that is currently on pip is outdated. I'm looking forward to releasing a new version soon (i.e, a month :\ ), and then I'll update on pip as well, and add a reference here.

Keeping this issue open until then.

**Comment by ggrrll** [comment]
ok thanks,

so for the time being better to install it  [this way](https://stackoverflow.com/a/15269110/5533078) I guess

**Comment by jaidevd** [comment]
Yep, I'll notify on this issue when I make a new release. I think I should also start a mailing list.

**Comment by jaidevd** [comment]
This will be fixed in the v0.2 release - targeted for end of February.
